src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenConditionalOperatorTests.cs

```diff
@@ -3025,25 +3034,118 @@ public static void Main()
 
             verify.VerifyIL("C<T>.Print()", @"
 {
-  // Code size       38 (0x26)
+  // Code size       62 (0x3e)
   .maxstack  2
-  .locals init (T V_0)
+  .locals init (T V_0,
+                T V_1)
   IL_0000:  ldarg.0
   IL_0001:  call       ""T C<T>.M()""
   IL_0006:  stloc.0
   IL_0007:  ldloca.s   V_0
-  IL_0009:  ldloc.0
-  IL_000a:  box        ""T""
-  IL_000f:  brtrue.s   IL_0015
-  IL_0011:  pop
-  IL_0012:  ldnull
-  IL_0013:  br.s       IL_0020
-  IL_0015:  constrained. ""T""
-  IL_001b:  callvirt   ""string object.ToString()""
-  IL_0020:  call       ""void System.Console.WriteLine(string)""
-  IL_0025:  ret
+  IL_0009:  ldloca.s   V_1
+  IL_000b:  initobj    ""T""
+  IL_0011:  ldloc.1
+  IL_0012:  box        ""T""
+  IL_0017:  brtrue.s   IL_002d
+  IL_0019:  ldobj      ""T""
+  IL_001e:  stloc.1
+  IL_001f:  ldloca.s   V_1
+  IL_0021:  ldloc.1
+  IL_0022:  box        ""T""
+  IL_0027:  brtrue.s   IL_002d
+  IL_0029:  pop
+  IL_002a:  ldnull
+  IL_002b:  br.s       IL_0038
+  IL_002d:  constrained. ""T""
+  IL_0033:  callvirt   ""string object.ToString()""
+  IL_0038:  call       ""void System.Console.WriteLine(string)""
+  IL_003d:  ret
 }");
+        }
+
+        [Fact, WorkItem(40690, "https://github.com/dotnet/roslyn/issues/40690")]
+        public void ConditionalAccess_GenericExtension_ValueTuple()
```

Comment:
>ConditionalAccess_GenericExtension_ValueTuple [](start = 20, length = 45)

Is there an equivalent test for VB that fails without the change?

Reply:
I think this test case would translate pretty straightforwardly into VB. I can add a regression test for it.